src/engines/basic-render-engine.ts

```diff
@@ -1,10 +1,10 @@
 import { EventEmitter } from 'events';
-import { deepMerge } from '../utils';
-import type { Dots, Mouse, RectRenderQueue, Stroke, Text, TooltipField } from '../types';
-import type { OffscreenRenderEngine } from './offscreen-render-engine';
+import { mergeObjects } from '../utils';
+import { Dots, Mouse, RectRenderQueue, Stroke, Text, TooltipField } from '../types';
+import { OffscreenRenderEngine } from './offscreen-render-engine';
 
 // eslint-disable-next-line prettier/prettier -- prettier complains about escaping of the " character
-const allChars = 'QWERTYUIOPASDFGHJKLZXCVBNMqwertyuiopasdfghjklzxcvbnm1234567890_-+()[]{}\\/|\'\";:.,?~';
+const allChars = 'QWERTYUIOPASDFGHJKLZXCVBNMqwertyuiopasdfghjklzxcvbnm1234567890_-+()[]{}\\/|\'";:.,?~';
```

Comment:
I assume this is autofix
not sure if it is correct to remove the `\` here?
if yes I guess we can remove the eslint-disable comment above

Reply:
It's just redundant escaping of `"` symbol, and the linter is right.)